package.json

```diff
@@ -50,6 +50,7 @@
   "dependencies": {
     "@bpmn-io/element-templates-validator": "^0.6.0",
     "@bpmn-io/extract-process-variables": "^0.4.5",
+    "@philippfromme/moddle-helpers": "^0.4.1",
```

Comment:
Related: Do you plan to maintain it for long? Otherwise it does make sense to migrate it over to `@bpmn-io`. We'd also need to verify that it is released under a liberal license, and built in a robust manner if we rely on it :see_no_evil:.

Reply:
I just made the repo public. Nothing to hide there. I would like to transfer this package to bpmn-io soon as we will need it in the future.